src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Helper/ResultHelper.cs

```diff
@@ -188,5 +188,52 @@ private static void SetScores(IEnumerable<Result> resultList, string query)
                 result.Score = lowScore--;
             }
         }
+
+        /// <summary>
+        /// Checks if a setting <see cref="WindowsSetting"/> matches the search string <see cref="Query.Search"/> to filter settings by settings path.
+        /// This method is called from the filter function <see cref="Main.Query(Query)"/> if the search string <see cref="Query.Search"/> contains the character ">".
```

Comment:
It's not directly `main.Query`. It's called from `predict()` method used in lynq query.